website/frontend/src/components/blocks/ComponentLoader/ComponentLoader.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from 'react';
+import { Loader } from 'react-feather';
+import styles from './ComponentLoader.module.scss';
+
+export const ComponentLoader: FC = () => {
```

Comment:
I think  color changing possibility is good for this component

Reply:
Also I believe `Spinner` is more declarative name since we already have `Loader`